version/constants.go

```diff
@@ -100,6 +100,13 @@ var (
 		constants.FujiID:    time.Date(2023, time.April, 6, 15, 0, 0, 0, time.UTC),
 	}
 	CortinaDefaultTime = time.Date(2020, time.December, 5, 5, 0, 0, 0, time.UTC)
+
+	// FIXME: update this before release
+	ContinuousStakingTimes = map[uint32]time.Time{
+		constants.MainnetID: time.Date(10000, time.December, 1, 0, 0, 0, 0, time.UTC),
+		constants.FujiID:    time.Date(10000, time.December, 1, 0, 0, 0, 0, time.UTC),
+	}
+	ContinuousStakingDefaultTime = time.Date(10000, time.December, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
keeping continuous staking fork disabled in local testnets too. Some e2e tests currently breaks. Will reactivate as soon as the tests are fixed.